### PR TITLE
fix: manage error while upload file

### PR DESCRIPTION
**Pull request by anistouri**
Signed-off-by: TOURI ANIS <anis-1.touri@rte-france.com>

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]
Shallow code read OK

**Comment by etiennehomer** [comment]
The CircularProgress stays displayed after the invalid file is uploaded. It shouldn't

File: src/components/dialogs/field-hook.js [hunk 1]
```diff
@@ -61,11 +61,13 @@ export const useTextValue = ({
     ...formProps
 }) => {
     const [value, setValue] = useState(defaultValue);
+    const [touched, setTouched] = useState(false);
```
Comment (etiennehomer) [inline src/components/dialogs/field-hook.js]: Maybe you could name this hook hasChanged, setHasChanged to be homogeneous with gridstudy-app

**Comment by etiennehomer** [comment]
And usePrefillNameField still tries to set the studyName to null when a invalid case is imported

**Comment by etiennehomer** [comment]
To me, there's something wrong in usePrefillNameField. Maybe you make the fix without adding the [touched, setTouched] hook but by fixing usePrefillNameField

File: src/components/dialogs/create-study-dialog.js [hunk 2]
```diff
@@ -281,12 +289,16 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
...
+        createStudyErr,
+        fileCheckedCase,
+        touched,
     });
 
     //Inits the dialog
     useEffect(() => {
         if (open && providedCase) {
-            setStudyName(providedCase.elementName);
+            // setStudyName(providedCase.elementName);
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/create-study-dialog.js]: remove this commented instruction

**Review by Walid-Sahnoun — APPROVED** [review]

**Pull request merged**